packages/core/src/util/html-encode.ts

```diff
@@ -0,0 +1,8 @@
+export function htmlEncode(str: string): string {
```

Comment:
nit: wouldn't it read better if we name this `encodeToHtml`?

Also, this is something that we could use even on the FE, so maybe store it on `packages/shared/src/common/html.ts`?